src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -254,6 +254,9 @@
     <PackageReference Include="NuGet.Packaging">
       <Version>5.0.0-preview1.5665</Version>
     </PackageReference>
+    <PackageReference Include="NuGet.Protocol">
+      <Version>4.8.0</Version>
```

Comment:
Align this version with NuGet.Packaging.

Reply:
done